drivers/modem/gsm_ppp.c

```diff
@@ -124,8 +159,14 @@ static void gsm_configure(struct k_work *work)
 
 		if (r < 0) {
 			LOG_DBG("modem setup returned %d, retrying...", r);
```

Comment:
As the strings are almost the same, we could save some flash space by writing these like this
```
		if (r < 0) {
			LOG_DBG("modem setup returned %d, %s", r, "retrying...");
		} else {
			LOG_DBG("modem setup returned %d, %s", r, "enable PPP");
			break;
		}
```

Reply:
Done.